cloud/amazon/ec2_eni.py

```diff
@@ -341,6 +355,37 @@ def get_sec_group_list(groups):
 
     return remote_security_groups
 
+def make_securitygroup_list(connection, module, group_list,subnet_id,eni_id):
```

Comment:
I've written a PR to give shared functionality for ec2 sec group name lookup https://github.com/ansible/ansible/pull/15054.  I'd rather use this than update the module as i like to keep things as DRY as possible.  Hopefully the PR will be merged soon and they i can make the small change in this module to use it